eos_data_distribution/ndn/base.py

```diff
@@ -118,9 +118,9 @@ def removeRegisteredPrefix(self, prefix):
 
     def registerPrefix(self, prefix=None, postfix="", flags=None):
         prefix = makeName(prefix)
-        postfix = makeName(postfix)
 
         if not prefix:
+            postfix = makeName(postfix)
```

Comment:
You can drop this entirely -- names will get constructed automatically if the thing you pass to `.append` isn't a Name already.